documentation/macros.sil

```diff
@@ -36,6 +36,7 @@
 \define[command=changed]{
 \note{\bf{The material in this section has changed significantly since the previous release of SILE.}}
 }
+\define[command=sup]{\raise[height=0.3em]{\font[size=0.8em]{\process}}}
```

Comment:
Not a big deal, but can we switch this to us `ex` units for the raise? Not a big deal here since we know the font, but as a demonstration if somebody copies this using a unit that is relative to font height rather than width is probably a better example to set.

Reply:
Oh, so _that's_ what `ex` is for :eyes: 

Sure, no problem.